src/alchemlyb/parsing/amber.py

```diff
@@ -164,31 +167,49 @@ def __init__(self):
 
 
 def file_validation(outfile):
-    """validate the energy output file """
+    """
+    Function that validate and parse an AMBER output file.
+    :exc:`ValueError` are risen if inconsinstencies in the input file are found.
+    
+    Parameters
+    ----------
+    outfile : str
+        Path to AMBER .out file to validate and extract data from.
+
+    Returns
+    -------
+    `:class:~FEData`
+        FEData object populated with data from the parsed AMBER output file.
+
+    """
+
     file_datum = FEData()
-    invalid = False
     with SectionParser(outfile) as secp:
         line = secp.skip_lines(5)
+
         if not line:
-            logger.warning('File does not contain any useful data, '
-                           'ignoring file.')
-            invalid = True
+            logger.error("the file does not contain any data, it's empty.")
```

Comment:
Let's add more information to the error log.
```suggestion
            logger.error("The file %s does not contain any data, it's empty.", outfile)
```